tests/pathfinding/fixtures/network_service.py

```diff
@@ -23,6 +24,11 @@
 from raiden_contracts.constants import CONTRACT_TOKEN_NETWORK_REGISTRY, CONTRACT_USER_DEPOSIT
 
 
+@pytest.fixture(scope="session")
+def pfs_address(create_service_account) -> Address:
```

Comment:
Why do we need this? We can always just read the address from the PFS object created by another fixture?